tests/template_backends/test_django.py

```diff
@@ -145,3 +146,32 @@ def test_non_debug_default_template_loaders(self):
     def test_debug_default_template_loaders(self):
         engine = DjangoTemplates({'DIRS': [], 'APP_DIRS': True, 'NAME': 'django', 'OPTIONS': {}})
         self.assertEqual(engine.engine.loaders, self.default_loaders)
+
+    @override_settings(DEBUG=True)
+    def test_debug_template_loaders_with_dirs(self):
+        engine = DjangoTemplates({
+            'DIRS': ['dirs'],
+            'APP_DIRS': True,
+            'NAME': 'django',
+            'OPTIONS': {},
+            'POST_APP_DIRS': ['post_app_dirs'],
+        })
```

Comment:
Yeah, I thought about this too. I agree with Tim that it's nice to have the form renderers depend on a consistent public API of all Django template engines, rather than having to make differing template-engine-specific workarounds. I don't feel strongly, though -- I agree that this is not likely to be a widely-used feature.